crates/live_kit_client/Cargo.toml

```diff
@@ -53,20 +55,23 @@ async-trait.workspace = true
 block = "0.1"
 bytes = "1.2"
 byteorder = "1.4"
-cocoa = "0.25"
-core-foundation = "0.9.3"
-core-graphics = "0.22.3"
 foreign-types = "0.3"
 futures.workspace = true
 hmac = "0.12"
 jwt = "0.16"
-objc = "0.2"
 parking_lot.workspace = true
 serde.workspace = true
 serde_derive.workspace = true
 sha2 = "0.10"
 simplelog = "0.9"
 
+
+[target.'cfg(target_os = "macos")'.dev-dependencies]
+core-foundation = "0.9.3"
```

Comment:
Hmm, I don't think we need `core-foundation` and `core-graphics` in `dev-dependencies`, as they're already included as normal dependencies. I know this isn't something introduced in this PR, but could we get rid of them while we're at it?

Reply:
In fact, none of these dependencies are used in the `dev` context. `core-graphics` are unused too; so we can get rid of this section + remove `core-graphics` dep.

Reply:
Sure. Fixed in [`3ab39d7` (#6854)](https://github.com/zed-industries/zed/pull/6854/commits/3ab39d7df9d88b1e327ab96fcf64367a8fce2a98)

Reply:
Nice! We can also get rid of core-graphics altogether and cocoa and objc in dev-dependencies.

Reply:
Here we go! 
[`30faa53` (#6854)](https://github.com/zed-industries/zed/pull/6854/commits/30faa538a482832364b21295d8a710584bfa7368)

Reply:
Cool, thanks; I hope I'm not too nitpicky, but live-kit-client is something that has pretty big impact on our clean build release times, so whatever can be done to make it build sooner is for the better. :)